documentation.html

```diff
@@ -278,7 +278,189 @@ <h1 class="heading--1-medium">Code</h1>
 						<!-- 	style="border: 0; transform: scale(1); overflow: hidden" -->
 					</div>
 				</section>
-				<!-- -----------BUTTONS--------- -->
+				<!-- --------------------------- -->
+				<div class="u-margin-bottom-medium"></div>
+				<hr />
+				<div class="u-margin-bottom-medium"></div>
+
+				<!-- -----------AVATAR--------- -->
+				<section class="doc-main-item" id="avatar">
+					<div class="main__header">
```

Comment:
Please format the code. Maintain consistent intentation throughout the code.

Reply:
Sure!